Runtime/Game/Requests/LeaderboardHistory.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using LootLocker.Requests;
+
+
+namespace LootLocker.Requests
+{
+    public class LootLockerLeaderboardHistoryResponse : LootLockerResponse
+    {
+        public LootLockerLeaderboardArchive[] archives { get; set; }
+    }
+    public class LootLockerLeaderboardArchive
```

Comment:
Should this contain a copy of the details about the leaderboard? Like the "has_metadata" field and the rest of that? How about reward information?